src/app/static/src/js/components/runReport/runReport.vue

```diff
@@ -154,14 +155,23 @@
                         this.defaultMessage = "An error occurred fetching Git commits";
                     });
             },
-            getParameterValues(values) {
+            getParameterValues(values, isValid) {
                 if (values) {
                     this.parameterValues.forEach((param, key) => {
                         if (values[key].name == param.name) {
                             param.value = values[key].value
                         }
                     })
                 }
+                this.validateParams(isValid)
+            },
+            validateParams(isValid) {
+                this.disableRun = false
+                this.paramError = ""
+                if (!isValid) {
+                    this.disableRun = true
+                    this.paramError = "Parameter value(s) required"
```

Comment:
Repetition from [this line](https://github.com/vimc/orderly-web/pull/303/files#diff-ebf72c62a1bcdbd89ecc04625319ce8a0af8b6a6518a3a568620971442fb760dR79)?